<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionSettingsDialog</class>
 <widget class="QDialog" name="ConnectionSettingsDialog">
  <property name="windowTitle">
   <string>Connection Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelProxyType">
       <property name="text">
        <string>Proxy Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="proxyTypeCombo">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HTTP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SOCKS5</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelHost">
       <property name="text">
        <string>Host:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="hostEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="portSpin">
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelUsername">
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="usernameEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelPassword">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="passwordEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>